composer.json

```diff
@@ -48,7 +50,7 @@
         "php": ">=5.6",
         "composer/installers": "~1.2.0",
         "vlucas/phpdotenv": "^2.0.1",
-        "johnpbloch/wordpress": "5.2.4",
+        "johnpbloch/wordpress": "*",
         "koodimonni-language/core-en_gb": "5.2.4",
```

Comment:
We will need koodimonni-language to have the same version as johnpbloch/wordpress - can you add the asterix please?

Reply:
Oh.
Asterix added.